azerothcore/azerothcore-wotlk
# fix(Core/Pets): Corrected pet speed catchup system.

Fixed #6098.



## Changes Proposed:
-  Ported formula from TC.

## Issues Addressed:

- Closes #6098
- Closes https://github.com/chromiecraft/chromiecraft/issues/716

## Tests Performed:

- Tested ingame with warlock's imp.

## How to Test the Changes:

1. Get to the mage tower in SW ,on the top of the platform that leads to mage trainers
2. Jump down and run to the trade distric in SW
3. The demon should run back the path and be with u by/near the time u get to the tailoring / herbalism trainer


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
